lib/publiccloud/eks.pm

```diff
@@ -15,6 +15,18 @@
 package publiccloud::eks;
 use Mojo::Base 'publiccloud::aws';
 use testapi;
+use utils;
+use publiccloud::k8s_utils;
+
+sub init {
+    my ($self) = @_;
+    $self->SUPER::init();
+
+    my $repository //= get_var("PUBLIC_CLOUD_CONTAINER_IMAGES_REPO", 'suse-qec-testing');
```

Comment:
I guess you don't need `\\` here.
Since $repository is not an argument, you declare it here as a variable and uses `get_var`, it will take either `PUBLIC_CLOUD_CONTAINER_IMAGES_REPO` or `suse-qec-testing`